src/DepartmentService.Broker/Consumers/GetDepartmentsConsumer.cs

```diff
@@ -31,7 +31,11 @@ private async Task<List<DepartmentData>> GetDepartmentsAsync(IGetDepartmentsRequ
         departmentsIds: request.DepartmentsIds,
         usersIds: request.UsersIds);
 
-      return dbDepartments.Select(_departmentDataMapper.Map).ToList();
+      Dictionary<Guid, List<Guid>> childDepartments = await _departmentRepository.GetChildIdsAsync(dbDepartments.Select(d => d.Id).ToList());
+
+      return dbDepartments.Select(d => _departmentDataMapper
+        .Map(d, childDepartments.TryGetValue(d.Id, out List<Guid> childIds) ? childIds : null))
```

Comment:
May be childrenIds

Reply:
yeah, sure! but it's no actual any more)